lib/pages/home/survey_page_viewer.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/pages/widgets/dimmed_background.dart';
+
+class SurveyPageViewer extends StatelessWidget {
+  final List<int> surveys;
+  final ValueNotifier<int> currentPageNotifier;
+  final _pageController = PageController();
```

Comment:
@luongvo we need to handle `dispose` for this controller manually 🙋 Or we can use the help from the lib `flutter_hook` that can handle the state of `controller`, `view state`, ...

Reply:
@manh-t noted 👌